build-system/server/new-server/transforms/scripts/scripts-transform.ts

```diff
@@ -36,6 +36,8 @@ function modifySrc(script: PostHTML.Node): PostHTML.Node {
 /**
  * Replace the src for every script tag to the local value.
  */
-export default function(tree: PostHTML.Node): void {
-  tree.match({tag: 'script'}, modifySrc);
+export default function(options: any): (tree: PostHTML.Node) => void {
+  return function(tree: PostHTML.Node){
```

Comment:
do we have a typescript linter?

Reply:
I don't think we do, but I found this:
https://github.com/typescript-eslint/typescript-eslint

Do we want to use it?

Reply:
yeah i think we do, separate task. just add space after the `)` to match other styles in the file